java/lang/Double/DoubleTest.cpp

```diff
@@ -762,41 +761,40 @@ TEST(JavaLang, DoubleByteValue) {
 	ASSERT_NOT_EQUAL(expectedResultByteValue , actualResultByteValue);
 }
 
-// TODO(thoangminh): Fix MAX_VALUE_DOUBLE, MIN_VALUE_DOUBLE , -0
+// TODO(thoangminh): Need to pass all test cases in comment block
 TEST(JavaLang, DoubleCompare) {
```

Comment:
ASSERT_EQUAL(expected, actual)

Reply:
I use ASSERT_NOT_EQUAL to test invalid case.

Reply:
i mean when you use ASSERT_EQUAL
expected result should be the first args
:1st_place_medal:

Reply:
Yep, got it